### PR TITLE
Update INSTALL.md

### PR DESCRIPTION
**Pull request by bkmgit**
Update Fedora dependencies following #3776

### CODE REVIEW

**Comment by Benau** [comment]
First you missed harfbuzz, and you don't need fontpackage to build the git stk-code

Also you don't need nettle anymore with openssl licensed change (or stk license change(soon will happen))

**Comment by bkmgit** [comment]
Thanks. Removed fontpackage. Can remove nettle, but this is listed as a main dependency and you indicate it will soon not be needed. Is it safe to remove it right now?

**Comment by Benau** [comment]
busy with something atm, tell u few days later

File: INSTALL.md [hunk 1]
```diff
@@ -30,10 +30,14 @@ To build SuperTuxKart from source, you'll need to install the following packages
 Fedora command:
```
Comment (Boloyxaya) [inline INSTALL.md]: xayavongbouasone1@gmail.com

Comment (Boloyxaya) [reply INSTALL.md]: xayavongbouasone1@gmail.com

**Comment by PramodhKumarM97** [comment]
good

**Comment by jaanajaanam** [comment]
Nice

**Comment by james165527** [comment]
Share codes please!

*Yours sincerely,*
Mark James Agripo.

**Comment by Benau** [comment]
Now it's default to openssl now (nettle in stk-code will soon be removed), you can adjust your pr accordingly.

Btw we will soon starting the dual-license of stk-code to gplv3+ and mpl2+, so it would be good if you can also agree the dual-licensing of your contributions...

**Comment by Muna58009924** [comment]
Hello friends,, my link a joined please
        https://www.hashingadspace.com/register.php?ref=Muna246

File: INSTALL.md [hunk 2]
```diff
@@ -30,10 +30,14 @@ To build SuperTuxKart from source, you'll need to install the following packages
...
+sudo dnf install @development-tools angelscript-devel \ 
+bluez-libs-devel cmake desktop-file-utils fontpackages-devel \
+freealut-devel freeglut-devel freetype-devel fribidi-devel \
+gcc-c++ git-core libXrandr-devel libcurl-devel libjpeg-turbo-devel \ 
+libpng-devel libsquish-devel libtool libvorbis-devel mesa-libEGL-devel \ 
+mesa-libGLES-devel openal-soft-devel openssl-devel libcurl-devel harfbuzz-devel \ 
+fribidi-devel mesa-libGL-devel libogg-devel libGLEW nettle-devel pkgconf \
+wiiuse-devel zlib-devel
 ```
 Mageia 6 command:
```
Comment (Fhad09111) [inline INSTALL.md]: Goood

**Comment by bkmgit** [comment]
Happy to agree to dual licensing of contributions. Is there a contributor licensing agreement to sign, or should this be part of the commit message? One of the builds seemed to fail because of network access issues, is it possible to re-trigger it. Assume it will be re-triggered on any updates.

**Comment by Benau** [comment]
edit your pr to include those statement? (see hows other pr was made)

**Comment by bkmgit** [comment]
# Agreement

By creating a pull request in stk-code, you hearby agree to dual-license your contribution as
GNU General Public License version 3 or any later version and
Mozilla Public License version 2 or any later version.

This includes your previous contribution under the same name of contributor.

Keep the above statement in the pull request comment for agreement.

**Comment by Benau** [comment]
btw i think you don't need nettle anymore (openssl now default)

**Comment by bkmgit** [comment]
Replaced Nettle with Openssl. See #4077

**Pull request closed without merging**